packages/core/src/components/SearchBox/SearchBox.stories.mdx

```diff
@@ -0,0 +1,30 @@
+import { SearchBox } from './SearchBox.tsx';
+import { Preview, Story, Meta, Props } from '@storybook/addon-docs/blocks';
+import * as stories from './SearchBox.stories';
+
+<Meta title="Core" component={SearchBox} />
+
+# SearchBox
+
+### Add a auto suggest text box
+
+<Preview withToolbar>
+    <Story name="SearchBox">{stories.Basic()}</Story>
```

Comment:
Move the definition over here only so that it can be visible in `show code` section

Reply:
Done